src/ashley/machina_extensions/forum_permission/handler.py

```diff
@@ -11,6 +12,28 @@
 class PermissionHandler(BasePermissionHandler):
     """Add ashley specific permission checks to the machina permission handler."""
 
+    def __init__(self):
+        """Constructor"""
+        super().__init__()
+        # This attribute is used to store the LTIContext of the current user.
+        # (see middleware.ForumPermission)
+        # It allows the permission checking functions to be aware of the current LTIContext
+        # we are scoped into, if any.
+        self.current_lti_context_id: Optional[int] = None
```

Comment:
is it really alway an integer? On all LMS?

Reply:
This is the primary key of our LTIContext model, so its always an integer.

The "context_id" you are referring to, that is transmitted in the LTI launch request, is actually a string, yes!

Reply:
IC :pray: